ChameleonForms.AcceptanceTests/PartialForTests.Should_render_correctly_when_used_via_form_or_section_and_when_used_for_top_level_property_or_sub_property.approved.html

```diff
@@ -0,0 +1,121 @@
+﻿Partials.cshtml

```

Comment:
This is the approval output from the acceptance test - it should be fairly self explanatory...